heralding/capabilities/http.py

```diff
@@ -65,11 +65,29 @@ def do_GET(self):
             headers_bytes = bytes(self.headers['Authorization'], 'utf-8')
             self.wfile.write(headers_bytes)
             self.wfile.write(b'not authenticated')
-
-    # Disable logging provided by BaseHTTPServer
+            aux_data = self.get_auxiliary_info()
+            self._session.add_auxiliary_info(aux_data)
+            # Disable logging provided by BaseHTTPServer
     def log_message(self, format_, *args):
         pass
 
+    def get_auxiliary_info(self):
+         data_fields = HTTPHandler.get_aux_fields()
+         data_list = {}
+         for field in data_fields:
+             if field in self.headers.keys():
+                 data_list.update({str(field) : str(self.headers[str(field)])})
+             else:
+                 data_list.update({str(field) : 'Header_Value_Not_Found.'})
+         data_list.update({'SessionID': str(self._session.id)})
```

Comment:
Why is it needed to set timestamp and sessionid? AFAIK they are already set in add_auxiliary_info...

Reply:
Yes, that is unnecessary. Is there anything else that I need to change?